src/common/Throttle.cc

```diff
@@ -368,3 +368,61 @@ void OrderedThrottle::complete_pending_ops() {
     ++m_complete_tid;
   }
 }
+
+uint32_t DynamicThrottle::calc_wait_time(uint32_t percentage_empty, 
+                  double &wait_time, bool return_start_delay_if_none) {
```

Comment:
Look at the style guide, out params *must* be pointers, and they must come last.